store/modules/flask/actions.js

```diff
@@ -24,6 +24,16 @@ export async function ping_system_status() {
     const data = result.data;
     const status_uuid = data.ui_status_code;
     const simulation_mode = data.in_simulation_mode;
+    if (data.plate_barcode != undefined) {
+      const plate_barcode = data.plate_barcode;
+      if (plate_barcode == "") {
+        this.commit("playback/set_barcode_number", null, { root: true });
```

Comment:
is `null` something that can be rendered correctly when directly tied to an `input` DOM element (in `PlateBarcode.vue`)? Does it show up as an empty input box, or does it have some undesirable behavior?

Reply:
Yes,  `null` is not rendered you can verify the same `pages/../../plate-barcode/basic.vue` shows up empty input box and the `barcode` value in Vuex of `playback` is set to `null` and no mutations are done in `basic.vue`, its captured in the screenshot `basic-init-base.png`